doc/sphinx/conf.py

```diff
@@ -67,8 +67,8 @@
     'trio': ('https://trio.readthedocs.io/en/stable/', None),
     'msal': ('https://msal-python.readthedocs.io/en/latest/', None),
     # Azure packages
-    'azure-core': ('https://azuresdkdocs.blob.core.windows.net/$web/python/azure-core/1.0.0/', None),
-    'azure-identity': ('https://azuresdkdocs.blob.core.windows.net/$web/python/azure-identity/1.0.0/', None),
+    'azure-core': ('https://azuresdkdocs.blob.core.windows.net/$web/python/azure-core/1.1.1/', None),
```

Comment:
If these are supposed to point to the latest release version we should come up with a better way keep these updated then manually doing.

Reply:
I do consider this to be somewhat on-demand, the intersphinx list is used to the `common` stuff, so they don't need to stay cutting edge all the time.

That being said, cannot disagree.